src/app/app.tsx

```diff
@@ -11,45 +14,44 @@ import { EditNonceDrawer } from '@app/features/edit-nonce-drawer/edit-nonce-draw
 import { IncreaseFeeDrawer } from '@app/features/increase-fee-drawer/increase-fee-drawer';
 import { Devtools } from '@app/features/devtool/devtools';
 import { jotaiWrappedReactQueryQueryClient as queryClient } from '@app/store/common/common.hooks';
-import { initSegment } from './common/segment-init';
+
 import { theme } from './common/theme';
+import { LoadingSpinner } from './components/loading-spinner';
 import { GlobalStyles } from './components/global-styles/global-styles';
 import { AppRoutes } from './routes/app-routes';
+import { persistor, store } from './store';
 
-const devToolsEnabled = false;
-
-declare global {
-  interface Window {
-    __APP_VERSION__: string;
-  }
-}
-
-window.__APP_VERSION__ = VERSION;
-
-void initSegment();
+const reactQueryDevToolsEnabled = false;
 
-export const App = () => {
+export function App() {
   return (
-    <ThemeProvider theme={theme}>
-      <GlobalStyles />
-      <QueryClientProvider client={queryClient}>
-        <ColorModeProvider defaultMode="light">
-          <>
-            <Router>
-              <AppErrorBoundary>
-                <AppRoutes />
-                <AccountsDrawer />
-                <NetworksDrawer />
-                <EditNonceDrawer />
-                <IncreaseFeeDrawer />
-                <SettingsDropdown />
-              </AppErrorBoundary>
-              <Toaster position="bottom-center" toastOptions={{ style: { fontSize: '14px' } }} />
-            </Router>
-            {devToolsEnabled && <Devtools />}
-          </>
-        </ColorModeProvider>
-      </QueryClientProvider>
-    </ThemeProvider>
+    <Provider store={store}>
+      <PersistGate loading={<LoadingSpinner />} persistor={persistor}>
+        <ThemeProvider theme={theme}>
+          <GlobalStyles />
+          <QueryClientProvider client={queryClient}>
+            <ColorModeProvider defaultMode="light">
+              <Suspense fallback={<LoadingSpinner />}>
```

Comment:
Should this be given a height? I am getting some weird loading spinner positioning at the very top of the page that seems a bit odd.

Reply:
Yeah I noticed this too, it used to be centred. Will fix.